app/controllers/concerns/mhv_controller_concerns.rb

```diff
@@ -38,7 +38,7 @@ def raise_requires_terms_acceptance
 
   def raise_something_went_wrong
     # TODO: any additional data could probably be provided in source.
-    raise Common::Exceptions::Forbidden, detail: 'Something went wrong. Please contact support'
+    raise Common::Exceptions::BackendServiceException, 'MHVAC1'
```

Comment:
Ideally this wouldn't be called from controller, but if you're trying to leverage i18n i guess for now this might be the easiest way. Alternately could create a custom exception class in Common::Exceptions which is what I think would be best to do for any exceptions raised in controllers.